### ISSUE TITLE
Set alarm state to "triggered" when alarm is activated?

### ISSUE THREAD

**Issue by shred86** [MisterWil/abodepy#46]
Is there any way to set the alarm state to "triggered" when the alarm is activated? I noticed it's being reported in the Home Assistant debug message:

2019-05-25 17:19:25 DEBUG (SocketIOThread) [abodepy.event_controller] Timeline event received: Front Door Interior Alarm Activated - Interior Alarm Activated (1132)

I'm just trying to figure out a way to set the alarm status to "triggered" in Home Assistant when the alarm is activated so you can setup automations such as turning the lights on when the alarm is going off.

**Comment by Tediore** [comment]
There's an abode_alarm event when the alarm triggers. I have an automation set up using this event. See https://www.home-assistant.io/components/abode/

**Comment by shred86** [comment]
Ah I forgot about those. That will work for now. I have all of my devices exposed to HomeKit where I have most of my automations setup. I was hoping to be able to create the automation through HomeKit based on the Abode Alarm status being changed to a triggered state. I suppose I could make a separate template switch based off the events but it's not a big deal. Supposedly Abode will have HomeKit support soon. :)

**Comment by Tediore** [comment]
Oh gotcha, makes sense. Yeah who knows when it'll be supported but they have said it will be.

**Comment by MisterWil** [comment]
Do Home-Assistant alarms have a "triggered" state now? So instead of watching for an alarm event you can instead just have an automation watching for a state change to the actual alarm entity?

**Comment by Tediore** [comment]
So apparently a triggered state does exist in the manual alarm control panel. Apparently though you can't set `to_state` as triggered, but you can use triggered in `from_state`
https://www.home-assistant.io/components/manual/

**Comment by shred86** [comment]
> Do Home-Assistant alarms have a "triggered" state now? So instead of watching for an alarm event you can instead just have an automation watching for a state change to the actual alarm entity?

Yeah, there's a `triggered` state for alarm control panels in Home Assistant. HomeKit also has a triggered state. I just manually set the alarm control panel state to `triggered` and it shows up as triggered in Home Assistant and HomeKit.

Not sure the most elegant way to implement this would be. Is it possible to just read the `event_code` from the JSON data packet that is pushed from the Abode servers to set the state from Abodepy to triggered (if event_code matches any value really unless we wanted to break it up by alarm types).
